classes/Tools.php

```diff
@@ -3051,31 +3049,28 @@ public static function getDirectories($path)
     /**
      * Return the directory list from the given $path using php glob function.
      *
-     * @param string $path
-     *
-     * @return array
+     * @return array<string>
      */
-    public static function getDirectoriesWithGlob($path)
+    public static function getDirectoriesWithGlob(string $path): array
```

Comment:
We  can't change signatures of public functions in minor version

Reply:
done